rozenborg/test-simulations-mgmt-repo
# Setting up project repository

#### What does this PR do?

* Setup repository for project 
* Automatic browser reloading upon changes in HTML, CSS and JS files.
* Set up configurations of HoundCI and ESLint

#### Description of Task to be completed?

* Set up a repository of project and showing at least one pull request
* Browser reloads when changes are made to either the html, css or js file
*  Shows ESLint Airbnb style violations in the terminal and HoundCI comment

#### How should this be manually tested?

To see basic repository setup with updated README.md go to the link
* https://github.com/andela-eshaibu/inverted-index/tree/dev

To test browser automatic reload run the following command on your terminal
* `git clone https://github.com/andela-eshaibu/inverted-index.git`
* `cd inverted-index`
* `npm install`
* `gulp`

To test ESLint Airbnb style violations in the terminal and HoundCI comment
```
git clone https://github.com/andela-eshaibu/inverted-index.git
cd inverted-index
git checkout chore/139967463/setup-linters-dev
npm install
./node_modules/.bin/eslint testlinters.js
```
#### Any background context you want to provide?

N/A
#### What are the relevant pivotal tracker stories?

* Setting up git repository
* Automate browser reloading
* Set up ESLint and HoundCI
#### Video

* Link to browser reloading screen cast
https://www.youtube.com/watch?v=GzsqLy9x150&t=3s

#### Screenshots (if appropriate)

* Eslint Airbnb violations in terminal
![eslint airbnd style violation](https://cloud.githubusercontent.com/assets/25608317/22990337/1eb199f6-f3b9-11e6-80db-5be0410dd23c.JPG)

* HoundCI comments on PR
![houndci comments](https://cloud.githubusercontent.com/assets/25608317/22991231/ee623942-f3bb-11e6-9559-f287f9c7dcbf.JPG)
#### Questions:

N/A
